docs/source/architecture/period_duration.rst

```diff
@@ -0,0 +1,61 @@
+Workers needed to make a per period (daily) on-chain commitment transactions indicating availability for the following
+period (after UTC midnight).
+
+The cost of the commitment transaction was approximately 200k gas but could cost more depending on the number of
+sub-stakes being staked. Given the significant increase in Ethereum gas since launch, the cost of the daily
+commitment transaction significantly increased and became untenable. Particularly for stake sizes closer to the minimum
+stake size, the NU staking rewards and fees received from running a Worker may not cover the cost of this commitment
+transaction.
+
+Reducing the transaction gas cost was a necessity, and there were multiple mitigation attempts:
+
+#. Removal of inactive sub-stakes i.e. sub-stakes that expired and were no longer relevant (`PR #2384 <https://github.com/nucypher/nucypher/issues/2384>`_).
+#. Addition of a Worker configuration setting to specify a cap on the gas price that Workers were willing to pay for the
+   commitment transaction. Commitment attempts are made throughout the current period, and if the gas price at the time
+   is too high, the commitment is not made (`PR #2445 <https://github.com/nucypher/nucypher/issues/2445>`_).
+
+Ultimately these were incremental solutions, which worked around the underlying issue. A more comprehensive next step
+was to increase the period duration i.e. reduce the frequency with which Workers needed to make the on-chain commitment
+transaction, thereby reducing the cost by a factor based on the increase in period duration.
+
+Taking a long-term perspective, lowering Worker overhead encourages more Stakers and Worker, and enables more
+competitive break-even policy fee price points for network usage which is likely to increase user adoption.
+
+
+7-Day Period
+------------
+
+A `proposal to increase the protocol period duration <https://dao.nucypher.com/t/1-improve-staker-p-l-by-increasing-period-duration/110>`_,
+outlining the pros and cons, was put forth to the :doc:`NuCypher DAO </architecture/dao>`, and a 7-day period duration
+received the majority of votes. On ``[some date]`` the NuCypher DAO approved the period migration, and subsequently,
+the period duration for the NuCypher protocol.
+
+Modifying the period duration to 7 days involved changes to the ``StakingEscrow`` and ``PolicyManager`` smart
+contracts, and the economics parameters for the protocol that include period duration, inflation parameters,
+minimum staking periods, policy fee per period etc. all of which needed to be adjusted due to the
+new minimum unit of time.
+
+The period duration modification reduces the frequency of on-chain
+commitment transactions (and therefore worker gas costs) and has minimal impact on the total staking rewards
+received for running a Worker.  However, since the commitment transaction occurs every 7 days, staking rewards
+will also be minted once every 7 days (instead of daily), but the sum minted will be 7 days worth.
```

Comment:
```suggestion
will also be minted once every 7 days (instead of daily), however the reward rate remains the same (the minted value will be 7 days worth).
```

Reply:
✔️